health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DependentFieldsWrapper.js

```
@@ -0,0 +1,964 @@
+                />
+            </div>
+
+            {/* Single-condition editor popup */}
+            {showPopUp && draftRule && (
+                <BodyPortal>
+                    <div className="popup-portal-overlay">
+                        <PopUp
+                            className="digit-popup--fullscreen"
+                            type={"default"}
+                            heading={editLabel}
+                            children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)